src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityContractProperty.cs

```diff
@@ -54,7 +55,7 @@ public SynthesizedRecordEqualityContractProperty(NamedTypeSymbol containingType,
 
         public override bool IsStatic => false;
 
-        public override bool IsVirtual => true;
+        public override bool IsVirtual { get; }
```

Comment:
I didn't understand that change. Seems like it should always be virtual, so that derived record types could override the equality contract.

On the other hand, this makes me wonder, should `IsSealed` below be based on whether the record type is sealed?

Reply:
Never mind, the PR description explain and clarifies ("IsVirtual and IsOverride cannot both be true"). Thanks!

---
In reply to: [436283471](https://github.com/dotnet/roslyn/pull/44912#discussion_r436283471) [](ancestors = 436283471)